### ISSUE TITLE
Have Speriment overwrite the title tag in exp.html

### ISSUE THREAD

**Issue by adamliter** [presleyp/Speriment#60]
In the README, you recommend not editing the `exp.html` file from psiTurk's temlpate files. However, this leaves the `<title>` tag unchanged. If one changes the `<title>` tags in the other files that you recommend editing (`ad.html`, `complete.html`, `consent.html`, `default.html` and `error.html`), then this will lead to inconsistent titles across the entire presented experiment.

A nice (though completely uncritical) enhancement would be to overwrite the `<title>` tag with the name of the experiment. In the interim, perhaps it is worth changing the README and suggesting that the `<title>` tag (and **only** the `<title>` tag) inside of `exp.html` should be edited.

**Comment by presleyp** [comment]
Good catch! People might want to title the page differently than they title the experiment in their script, so I think I'll go with the README update. Eventually it would be nice to streamline those changes...maybe some bash commands like `title "My Experiment`.

**Comment by adamliter** [comment]
That's true. Yeah, I'm not sure what the best option would be. Specifying two different titles inside the Speriment script seems like it would be needlessly confusing, for example. But maybe a default option that puts the name of the Speriment experiment inside the `<title>` tag of the `exp.html` makes sense? I'm not really sure.
